app/components/works/dates_component.rb

```diff
@@ -57,15 +57,15 @@ def created_approximate?
     delegate :published_edtf, to: :reform
 
     def created_range_start_year
-      created_range_start&.year
+      created_range_start&.year || reform.send(:'created_range(1i)').presence&.to_i
```

Comment:
So that the original submitted values are returned when an EDTF couldn't be created.

Reply:
Can you add that as a comment in the code?

Reply:
Added